### ISSUE TITLE
moveToFile does not combine imports 

### ISSUE THREAD

**Issue by iisaduan** [microsoft/TypeScript#57257]
### 🔎 Search Terms

moveToFile, multiple auto imports, import order

### 🕗 Version & Regression Information

This is the behavior in every version I tried

### ⏯ Playground Link

No playground for moveToFile

### 💻 Code

```
// @Filename: /bar.ts
import { a, z } from './a';
const q = 0;
```
```
// @Filename: /a.ts
import { } from './other';
import type { } from './other';

export const p = 0;
export const b = 1;
export const a = 2;
export const z = 3;
[|const y = p + b;|]
```
Move the selection in `a.ts` to `bar.ts`

### 🙁 Actual behavior

A separate import statement is generated.
```
//// @Filename: /bar.ts
import { p, b } from './a';
import { a, z } from './a';
const q = 0;
const y = p + b;
```

### 🙂 Expected behavior

The named imports should combine into one statement: 
```
// @Filename: /bar.ts
import { a, b, p, z } from './a';
const q = 0;
const y = p + b;
```

### Additional information about the issue

I found this while working with auto-imports and playing with actions that generate multiple imports. The ideal order of imports should be `a, b, p, z`, as the imports in the original file were sorted.  From https://github.com/microsoft/TypeScript/issues/33839#issuecomment-642294737, and the current behavior of the `add all missing imports` codefix, `p, b` should be generated only if the original imports are not sorted.

**Comment by DanielRosenwasser** [comment]
Duplicate of #54309?

**Comment by iisaduan** [comment]
@DanielRosenwasser Not quite, the imports here should be added, they just generate in a weird order. Its possible the fix for both is related.

**Comment by RyanCavanaugh** [comment]
Closing language service bugs related to the 6.0 implementation. For more information, see #62827
